### DISCUSSION TITLE
importing a filter list does not work

### DISCUSSION THREAD

**Discussion by ghajini** [MasterKia/PersianBlocker#224] [Q&A]
@MasterKia 
As title suggests , when i add https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlocker.txt
to import list in ublock origin settings, it does not work
ubo 1.56, edge

**Comment by MasterKia** [comment]
@JobcenterTycoon Sorry, can you reprouduce too?

**Reply by JobcenterTycoon** [reply]
Yes on FF

**Reply by ghajini** [reply]
chrome also affected

**Comment by ghajini** [comment]
i added other lists such ublock filters or
eg https://raw.githubusercontent.com/easylist/EasyListHebrew/master/EasyListHebrew-uBO.txt
 then above gets added
but when i import below list, it never gets added
 https://raw.githubusercontent.com/MasterKia/PersianBlocker/main/PersianBlocker.txt

chrome/edge/firefox
ubo 1.56

**Comment by MasterKia** [comment]
I can reproduce on Firefox mobile (uBO 1.56.0).

@gorhill Sorry, could you take a look?

**Comment by gorhill** [comment]
If you select another pane then come back to "Filter lists", "PersianBlocker" in stock lists will be properly selected. There is an issue when importing a URL which is already available through stock lists.
